src/app/externs/backendapi.js

```diff
@@ -1182,6 +1184,7 @@ backendApi.SecretList;
  * @typedef {{
  *   objectMeta: !backendApi.ObjectMeta,
  *   typeMeta: !backendApi.TypeMeta,
+ *   secretType: string
```

Comment:
What is this? You modified `IngressDetail`, so is not related at all. It looks like you are pasting random code snippets right now...